src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        truck.setName("Truck");
+        Machine bulldozer = new Bulldozer();
+        bulldozer.setName("Bulldozer");
+        Machine excavator = new Excavator();
+        excavator.setName("Excavator");
```

Comment:
Could you init name by constructor instead of setter

Reply:
done